src/main/resources/templates/item.html

```diff
@@ -20,15 +20,14 @@
 
       <div class="grid-row">
         <div class="column-two-thirds">
-          <p></p>
+          <h1 class="heading-large">Item</h1>
```

Comment:
This isn't very descriptive for assistive technologies but I don't have a better idea about what to say instead, so let's leave it like this for now.

(Lots of screen readers allow you to press `1` to go to the H1 on the current page, and when read out, "Item" isn't very useful)